tests/person_detection.robot

```diff
@@ -28,7 +28,7 @@ Run Test
     Execute Command                 Clear
     Execute Command                 mach create
     Execute Command                 machine LoadPlatformDescription @platforms/boards/arduino_nano_33_ble.repl
-    Execute Command                 sysbus LoadELF @${CURDIR}/binaries/person_detection.ino.elf
+    Execute Command                 sysbus LoadELF @${CURDIR}/binaries/person_detection/person_detection.ino.elf
```

Comment:
Let's reorganize the structure.

Instead of 
-> tensorflow
-> notebooks
-> tests

Let's have
-> tensorflow
-> person-detection
->-> ipynbs
->-> .robot
->-> buildscript
->->etc
-> microspeech
->->...

Reply:
Also, we have a readme - it needs to be updated accordingly

Reply:
I'll do that in the next PR